src/html/rangeTemplate.html

```diff
@@ -1,16 +1,27 @@
 <p class="gpiic-fd-range-instructions gpii-fd-instructions"></p>
 <div class="row gpii-fd-controls">
-    <div class="small-offset-1 small-1 medium-offset-4 columns gpii-fd-status">
-        <div class="gpiic-fd-range-indicator gpii-fd-statusFill"></div>
+    <div class="row">
+        <div class="small-offset-1 small-1 medium-offset-3 columns">max</div>
     </div>
-    <div class="small-7 medium-6 end columns">
-        <div class="row">
-            <input type="button" value="+" class="gpiic-fd-range-increase left gpii-fd-increase" id="gpiic-fd-increase" tabindex="1" />
-            <label class="gpiic-fd-range-increaseLabel gpii-fd-control-label" for="gpiic-fd-increase"></label>
+    <div class="row">
+        <div class="small-offset-1 small-1 medium-offset-4 columns gpii-fd-status">
+            <div class="gpii-fd-statusFillMin"></div>
+            <div class="gpii-fd-statusFillContainer">
+                <div class="gpiic-fd-range-indicator gpii-fd-statusFill"></div>
+            </div>
         </div>
-        <div class="row">
-            <input type="button" value="-" class="gpiic-fd-range-decrease left gpii-fd-decrease" id="gpiic-fd-decrease" tabindex="2" />
-            <label class="gpiic-fd-range-decreaseLabel gpii-fd-control-label" for="gpiic-fd-decrease"></label>
+        <div class="small-7 medium-6 end columns">
+            <div class="row">
+                <input type="button" value="+" class="gpiic-fd-range-increase left gpii-fd-increase" id="gpiic-fd-increase" tabindex="1" />
```

Comment:
In the iOS simulator in Xcode, the + and - characters are not properly centered in the container. Can you confirm that it looks fine on a real device?